presto-docs/src/main/sphinx/release/release-0.193.rst

```diff
@@ -0,0 +1,71 @@
+=============
+Release 0.193
+=============
+
+General Changes
+---------------
+
+* Fix an infinite loop during planning for queries containing non-trivial predicates.
+* Fix a bug that causes queries that constrain the result of ``row_number()`` to numbers less than or
+  equal to 0 to fail or return incorrect results.
+* Fix failure during query planning when lambda expressions are used in ``UNNEST`` or ``VALUES`` clauses.
+* Fix ``Tried to free more revocable memory than is reserved`` error for queries that run in the
+  reserved memory pool with spilling enabled.
+* Improve the performance of the :func:`ST_Contains` function.
+* Add :func:`map_zip_with` lambda function.
+* Add :func:`normal_cdf` function.
+* Add ``SET_DIGEST`` type and related functions.
+* Add query stat to keep track of peak total memory.
+* Improve performance of queries that filter all data from a table up-front (e.g., due to partition pruning).
+* Turn on new local scheduling algorithm by default (see :doc:`release-0.181`)
+* Remove the ``information_schema.__internal_partitions__`` table.
+
+Security Changes
+----------------
+
+* Apply the authentication methods in the order they are listed in the
+  ``http-server.authentication.type`` configuration.
+
+CLI Changes
+-----------
+
+* Fix rendering of maps of Bing tiles.
+* Abort the query correctly when the result pager exits.
+
+JDBC Driver Changes
+-------------------
+
+* Use SSL by default for port 443.
+
+Hive Changes
+------------
+
+* Fix a bug that causes a ``ConcurrentModificationException`` when dropping any column
+  other than the last one.
+* Fix a bug that causes a newly added partition to have incorrect on-disk data format
+  for tables using the text file format.
+* Prevent queries from failing when they read data from regions other than ``us-east-1``.
+  Previously, such queries would fail with
+  ``"The authorization header is malformed; the region 'us-east-1' is wrong; expecting '<region_name>'"``,
+  where ``<region_name>`` is the S3 region hosting the bucket that is queried.
+* Enable ``SHOW PARTITIONS FROM <table> WHERE <condition>`` to work for tables
+  that have more than ``hive.max-partitions-per-scan`` partitions as long as
+  the specified ``<condition>`` reduces the number of partitions below this limit.
+
+Blackhole Changes
+-----------------
+
+* Prevent creating tables in a nonexistent schema.
+* Add support for ``CREATE SCHEMA``.
+
+Memory Connector Changes
+------------------------
+
+* Fix ``ALTER TABLE ... RENAME TO ...`` to use the destination schema if the destination schema
+  is different than the source schema.
+* Prevent creating tables in a nonexistent schema.
+
+MongoDB Changes
+---------------
+
+* Fix ``INSERT INTO`` support, which was broken since the removal of the legacy connector API in :doc:`release-0.155`.
```

Comment:
Only link to previous release notes if there's a direct reason someone would want to read it (such as for the scheduling algorithm change as above, since it is discussed in those release notes). In this case, there's nothing about this removal / breakage in those release notes.

The command is named `INSERT`, not `INSERT INTO` (see title of the command in the manual).

I would avoid saying "fix" and "broken" here since those are ambiguous. They could mean it inserts partial results, fails for certain data types, etc.

```
* Add ``INSERT`` support. It was previously removed in 0.155.